### ISSUE TITLE
gstreamer Value not found in <Enumeration> 'TriggerSelector'

### ISSUE THREAD

**Issue by lc-spxl** [AravisProject/aravis#702]
Hi,
first of all thank you for this great project! It's amazing!

I am trying to start a pipeline on a HIKROBOT camera

The pipeline is like this
`aravissrc camera-name=Hikrobot-MV-CA032-10GC-02K32049763 ! video/x-raw, format=RGB,width=640,height=480 ! jpegenc idct-method=2 quality=100 ! multipartmux boundary="myboundary" ! tcpserversink sync=false host=0.0.0.0 port=8000`

Unfortunately I am hitting this error 

```
Setting pipeline to PAUSED ...
Pipeline is live and does not need PREROLL ...
Pipeline is PREROLLED ...
Setting pipeline to PLAYING ...
ERROR: from element /GstPipeline:pipeline0/GstAravis:aravis0: Could not set caps on camera "Hikrobot-MV-CA032-10GC-02K32049763": Value not found in <Enumeration> 'TriggerSelector'
```

I checked the issues history and also tried to get a clue from the code (I have a very basic experience with gst plugin)

Do you have an idea why this error occurs ? 

Thank you

**Comment by EmmanuelP** [comment]
Hi,

Which version of Aravis are you using ? This issue should be fixed in the latest version, 0.8.22.

**Comment by lc-spxl** [comment]
I downloaded `0.8.9` but also just built against `main` branch with the same result
